pkg/memory/repository.go

```diff
@@ -0,0 +1,69 @@
+package memory
+
+import (
+	"fmt"
+	"github.com/goharbor/harbor-scanner-clair/pkg/job"
+	"github.com/goharbor/harbor-scanner-clair/pkg/model/harbor"
+	"sync"
+)
+
+type repository struct {
```

Comment:
Why name it `repository`?

Reply:
The repository pattern is quite common in DDD & hexagonal architecture projects. It's an interface abstracting access to domain entities. Anyway I renamed it to `persistence.Store`, otherwise it might be confusing with Docker repository.